truenas_install/__main__.py

```diff
@@ -111,7 +117,7 @@ def install_grub_freebsd(input, manifest, pool_name, dataset_name, disks):
             """
         else:
             efi_partition_uuid = run_command([
-                "grub-probe", "--device", f"/dev/{disks[0]}p1", "--target=fs_uuid"
+                "grub-probe", "--device", get_partition(disks[0], 1), "--target=fs_uuid"
```

Comment:
Since you're using a bug tracker and assigning priority: this line as originally written only allowed TrueNAS to install on EFI systems with NVME installation target drives.

This change also allows TrueNAS to be installed on EFI systems onto HDDs and SATA SSDs (including M.2 SATA drives).